src/components/App/AppDriver.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  DropdownTestkit,
+  inputTestkitFactory,
```

Comment:
Try not to use `*TestkitFactory` driver. The correct way to use it is with `*Testkit`.
```suggestion
  InputTestkit,
```